homeassistant/components/motionmount/__init__.py

```diff
@@ -0,0 +1,48 @@
+"""The Vogel's MotionMount integration."""
+from __future__ import annotations
+
+import motionmount  # type: ignore[import-untyped]
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST, CONF_PORT, Platform
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+from .coordinator import MotionMountCoordinator
+
+PLATFORMS: list[Platform] = [
+    Platform.NUMBER,
+]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Vogel's MotionMount from a config entry."""
+
+    hass.data.setdefault(DOMAIN, {})
+
+    # Create API instance
+    coordinator = MotionMountCoordinator(hass)
+    mm = motionmount.MotionMount(
+        entry.data[CONF_HOST], entry.data[CONF_PORT], coordinator.motionmount_callback
+    )
+    coordinator.mm = mm
```

Comment:
I don't really like the way this is set up. Can the MotionMount lib maybe create a function to add a listener afterwards instead of during construct?